Docs/Book/C++Examples/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required( VERSION 2.8.9 )
+
+project(RDKitSV)
+
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}")
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -std=c++11" )
+
+set(Boost_USE_STATIC_LIBS ON)
+set(Boost_USE_MULTITHREADED OFF)
+set(Boost_USE_STATIC_RUNTIME ON)
+find_package( Boost COMPONENTS iostreams filesystem system thread)
+
+find_package( Cairo REQUIRED )
```

Comment:
Yes.  I thought it would be useful to include as well as SVG, as it works somewhat differently.  I assume that when I get a proper CMakeLists.txt organised, I can put guards round it as you do in the relevant test1.cpp.